crates/bitwarden-crypto/src/keys/master_key.rs

```diff
@@ -118,39 +124,44 @@ impl MasterKey {
     }
 
     pub fn to_base64(&self) -> String {
-        self.0.to_base64()
+        STANDARD.encode(self.0.key_material.to_vec())
```

Comment:
I think to_vec creates an additional copy on the heap, do we need to zeroize explicitly?

Reply:
Can be replaced by `as_slice`